spec/features/user_can_view_categories_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.feature 'User can view categories' do
+  context 'There are categories' do
+    scenario 'User visits /categories' do
+      c1, c2 = create_list(:category, 2)
+
+      visit categories_path
+
+      expect(current_path).to eq '/categories'
+
+      within('ul.categories') do
```

Comment:
This is cool. I think we'll probably want to test for the presence of creatures belonging to those categories, too. I'm picturing the html relationship will look something like, "ul.categories li.category" And then the row of cards we were discussing living in there. That might be a different feature test, though.